torch/csrc/jit/passes/quantization.cpp

```diff
@@ -986,10 +986,20 @@ void InsertObserversHelper::preprocess(
   }
 }
 
-// Returns true if the value is the weight to LSTM operator.
-bool isDynamicLSTMWeight(Value* v, Use use, bool is_dynamic) {
-  return is_dynamic && use.user->kind() == Symbol::aten("lstm") &&
-      (use.offset == 2);
+bool valueQuantizable(Use use, bool is_dynamic) {
+  Node* n = use.user;
+  // static quantization
+  if (n->kind() == prim::CallFunction &&
+      getFuncName(n->inputs()[0]) == "batch_norm") {
+    return use.offset == 1;
+  }
+  // Dynamic quantization checks
+  if (is_dynamic) {
+    if (use.user->kind() == Symbol::aten("lstm")) {
+      return use.offset == 2;
+    }
+  }
+  return nodeQuantizable(n);
```

Comment:
I feel hiding this here is not very explicit, I think moving this outside and use `nodeQuantizable && useQuantizable` is probably better